package.json

```diff
@@ -12,9 +12,9 @@
     "start": "npm run build -- -w",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "test:all": "npm run test:coverage && npm run test:lint",
-    "test": "jest",
-    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
-    "test:watch": "jest --watch",
+    "test": "jest --env node",
```

Comment:
Actually, Jest uses less memory than Karma does, by quite a bit. That said, this change if needed belongs in `webpack-defaults`.

All of our libs run the exact same projects setup. So for the scope of this pull request, this needs to be set back to what it was. 

If you feel that strongly about this particular configuration, open a pull request in https://github.com/webpack-contrib/webpack-defaults/issues so the change can be applied to all of our libs in the next patch release of defaults.